docs/commerce/checkout/configure_checkout.md

```diff
@@ -0,0 +1,195 @@
+---
+description: Configure the default checkouts process and build custom ones.
+edition: commerce
+---
+
+# Configure checkout
+
+When you work with your Commerce implementation, you can review and modify 
+the checkout configuration.
+
+!!! note "Permissions" 
+
+    When you modify the workflow configuration, make sure you properly set user [permissions](permission_use_cases.md#commerce) to the checkout component.
+
+## Checkout workflow
+
+Checkout workflow is configured like other [workflows](workflow.md), and each 
```

Comment:
If this refers to Ibexa Workflow component, this sentence is wrong. In checkout, we rely only on **Symfony** Workflow. No configuration from our Workflow Component can be used - it works only for content objects wheras checkout-specific objects are not content per se.

Reply:
I have removed a reference to our workflow